src/CMakeLists.txt

```diff
@@ -295,8 +295,7 @@ if(BUILD_AVDECC_LIB_SHARED_CXX)
 
 	# Setup libfmt
 	if(ENABLE_AVDECC_USE_FMTLIB)
-		target_compile_options(${PROJECT_NAME}_cxx PRIVATE -DHAVE_FMT)
-		target_link_libraries(${PROJECT_NAME}_cxx PRIVATE fmt-header-only)
```

Comment:
I don't understand how that change would work better than using the fmt author provided header-only target. Linking with it guarantees everything is correctly setup by the author. I actually don't see how it can work without specifying the location of the header file. I suspect it would fall in a working state because cmake automatically adds the fetched content as a "-I" path. But if the author then decides to change the internal define (ie. FMT_HEADER_ONLY) or add a new one, it will not be passed to the compilation units if not using the defined target.

Reply:
Oh, it failed to find fmt-header-only and somehow I didn't find it upstream either. I probably was too tired. It looks like I just have to change fmt-header-only to fmt::fmt-header-only. I'll change that.